platforms/stm32_common/usb/hid.c

```diff
@@ -8,7 +8,7 @@
 #include "platform/usb/usb.h"
 #include "usb/core.h"
 #include "usb/ctlreq.h"
-#include "stm32f0xx_hal.h"
+#include "stm32_hal.h"
 #include "hid_internal.h"
```

Comment:
:<

Reply:
sorting breaks the build

Reply:
Fix it then :P

Reply:
no but seriously, i'd rather spend my time fixing bugs, improving stuff. if you care so much about sorting the includes, why don't you make a pr :)

Reply:
Bad @kbeckmann